db/migrate/20210317090500_add_disks_total_to_host_facets.rb

```diff
@@ -0,0 +1,9 @@
+class AddDisksTotalToHostFacets < ActiveRecord::Migration[6.0]
+  def up
+    add_column :host_facets_reported_data_facets, :disks_total, :integer
```

Comment:
What's the largest total disks that we can store in an integer? is that enough or do we need a `bigint`?